libqtile/drawer.py

```diff
@@ -162,7 +168,27 @@ def draw(self, x, y, rounded=True, fill=False):
             self.layout.height + self.pad_top + self.pad_bottom,
             self.border_width
         ]
-        if fill:
+        if line:
+            highlight_opts = [
+                x, 0,
+                self.layout.width + self.pad_left + self.pad_right,
+                bar_height,
+                self.border_width
+            ]
+            self.drawer.set_source_rgb(self.highlight_color)
+            self.drawer.fillrect(*highlight_opts)
+
+            lineopts = [
+                x,
+                bar_height - self.line_thickness,
```

Comment:
I'd say probably use the height of the frame, i.e. `self.layout.height + self.pad_top + self.pad_bottom`